tests/apps/test_sc.py

```diff
@@ -11,7 +11,7 @@ def test_design_inference(scroot, monkeypatch):
     source = os.path.join(scroot, 'tests', 'data', 'heartbeat.v')
     # only run import, makes this quicker
 
-    monkeypatch.setattr('sys.argv', ['sc', source, '-steplist', 'import', '-strict'])
+    monkeypatch.setattr('sys.argv', ['sc', source, '-from', 'import', '-strict'])
```

Comment:
Should this be `-to import` instead? (won't break test either way, but should make it faster)